src/r_bsp.c

```diff
@@ -372,6 +395,8 @@ static void R_AddLine (seg_t *line)
   if (x1 >= x2)       // killough 1/31/98 -- change == to >= for robustness
     return;
 
+  R_MaybeInterpolateTextureOffsets(line->sidedef);
```

Comment:
The only thing I'm not sure about is where to call `R_MaybeInterpolateTextureOffsets`. Other than that, I think it's ready to merge.

Reply:
Looks good to me.